ProfileView/src/index.ts

```diff
@@ -76,7 +76,7 @@ export default class CjaasProfileWidget extends LitElement {
     | null = null;
   /**
    * SAS Token for POST operations on Profile endpoint
-   * @attr profile-write-token
+   * @attr profile-token
```

Comment:
Why are we getting rid of permission info from attribute name. It would explicitly inform user on what token to use.

Reply:
So what's your opinion, should the widget be so explicit as to have a specific token for read and write? Or rely upon docs to tell users that this widget needs both permissions, and allow that to happen with one token . . . then we have one token to rule them all, and confusion creeps back in when a user wonders if 'profile-write-token' automatically implies that it reads as well.

So long story short, I prefer "provide one token" and docs say "make the token read and write"